File: src/Basic_DSA/Prime_Number.java
package Basic_DSA;

public class Prime_Number {
    static void main(String[] args) {
        int n = 1;
        boolean isPrime = true;
        if(n <= 1) {
            System.out.println("Not Prime");
            return;
        }
        for(int i = 2; i <= Math.sqrt(n); i++) {
            if(n % i == 0) {
                isPrime = false;
                break;
            }
        }
        if(isPrime) {
            System.out.println("True");
        } else {
            System.out.println("False");
        }
    }
}

File: Python/Basics/prime_number.py
import math
num = int(input('Enter a number: '))
if num <= 1:
    print('Number is not a Prime Number')
else:
    is_Prime = True
    for i in range(2, int(math.sqrt(num)) + 1):
        if num % i == 0:
            is_Prime = False
            break
    if is_Prime:
        print('Number is a Prime Number')
    else:
        print('Number is not a Prime Number')